src/main/java/lithium/university/TwitterPublish.java

```diff
@@ -24,12 +25,18 @@ public static void main(String args[]){
         }
     }
 
-    public void postToTwitter(Twitter twitter, String message) throws TwitterException {
+    /*
+    * Takes a message and error checks it before attempting to post to user's twitter
+    * Input: Twitter twitter - twitter instance, String message - message to be posted
+    * Output: true if successful, false otherwise
+    * */
+    public boolean postToTwitter(Twitter twitter, String message) throws TwitterException {
```

Comment:
you're already using exception for the updateStatus call... any reason to not do something similar with your check?